### DISCUSSION TITLE
is there any slideshow functionallity given ?

### DISCUSSION THREAD

**Discussion by elschman** [LycheeOrg/Lychee#1839] [Q&A] [answered]
n.t.

**Comment by ildyria** [comment, marked as answer]
https://github.com/LycheeOrg/Lychee/issues/949
https://github.com/LycheeOrg/Lychee/issues/1814

> Thanks to @mingan666 with #1819 this is now supported in the `/frame` mode. You can specify which album to display in parameters.

**Reply by elschman** [reply]
how to specify the album ? The name ? The nmber (where to get ?)

**Reply by mingan666** [reply]
Maybe the answers here will help you: https://github.com/LycheeOrg/Lychee/issues/1814
